src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java

```diff
@@ -274,12 +273,47 @@ private void verifyConsistence(String id, List<PaymentJob> paymentJobs, Map<Stri
         }
     }
 
+    private CompletableFuture<Boolean> notifyWorker(PaymentJob paymentJob) {
+        int length = paymentJob.getMessage().length();
+        int current_location = 0;
+        int counter = 0;
+        List<CompletableFuture<Boolean>> messages = new ArrayList<>();
+
+        String tooLongMessage = loadFiles("/mturk/TooLongMessage.txt");
+        String subjectLine = loadFiles("/mturk/SubjectLine.txt");
+        //while loop sents messages if needed message would be gibber than MAX_LENGTH
+        while(length - current_location > NotifyWorker.MAX_LENGTH) {
+            int old_current_location = current_location;
+            //the new location is the maximum - the too long message
+            current_location = current_location + (NotifyWorker.MAX_LENGTH - tooLongMessage.length());
+
+            String message = paymentJob.getMessage()
+                    .substring(old_current_location, current_location) + "\n"+tooLongMessage;
```

Comment:
Because i need to split the string into parts which are fitting into the message ...